aws/aws_kms_test.go

```diff
@@ -42,31 +60,26 @@ func NewAwsSecretTest(secretConfig map[string]interface{}) (test.SecretTest, err
 func (a *awsSecretTest) TestPutSecret(t *testing.T) error {
```

Comment:
so now we are not testing file persistence store?

Reply:
we are testing filePersistenceStore. I have removed the part where we used to pass in the cipherBlob itself in the API and then compare what was returned from KMS.